software/tests/experimental/test_knobs.py

```diff
@@ -258,9 +258,9 @@ def test_access_by_index(mockHardware: MockHardware, knob_bank: KnobBank):
     mockHardware.set_ADC_u16_value(knob_bank.current, MAX_UINT16 / 5)
 
     assert round(knob_bank.knobs[0].percent(), 2) == 0
-    assert round(knob_bank.knobs[1].percent(), 2) == 0.80
+    assert round(knob_bank.knobs[1].percent(), 2) == 0.81
```

Comment:
[required] Rather than just changing the values in these tests (as well as those in test_knob.py) I would duplicate them and test both with and without a deadzone. So the existing tests could remain to test `deadzone=0.0` and the new tests with the new values would test `deadzone=0.01` (or whatever the default ends up being at the end of this PR).

Doing it this way does two things: First it shows that this PR doesn't affect the old behavior, which is still available with `deadzone=0.0`. Second, it serves as a form of documentation, showing exactly how we expect the behavior to change when the deadzone feature is used.